### PR TITLE
Mixpanel source updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by a-rampalli — APPROVED** [review]
lgtm

File: docs/sources/extract/mixpanel.mdx [hunk 1]
```diff
@@ -28,22 +28,15 @@ To set up Mixpanel as a source in RudderStack, follow these steps:
...
 
-- **Project Token** : Enter your Mixpanel project token by going to your Mixpanel **Project Settings** > **Access Keys**.
+- **Authentication**: Select either **Service Account** or **Project Secret** from the dropdown to give RudderStack the required permissions to access your Mixpanel account. Refer to the [Service Accounts](https://developer.mixpanel.com/reference/service-accounts) and [Project Secret](https://developer.mixpanel.com/reference/project-secret) documentation for more information on obtaining these details.
 
-<div class="infoBlock">
-For more information on obtaining the Mixpanel project token, refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004502806-Find-Project-Token-">Mixpanel support guide</a>.
-</div>
+<img src="../../assets/cloud-extract-sources/mixpanelv2-connection-settings.png" alt="Mixpanel connection settings" />
 
+- **Project ID** : Enter your Mixpanel project ID by going to your Mixpanel **Project Settings** > **Overview**. Refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id">Mixpanel support guide</a> for more information.
```
Comment (ameypv-rudder) [inline docs/sources/extract/mixpanel.mdx:37]:
```suggestion
- **Project ID** : Enter your Mixpanel project ID by going to **Project Settings** > **Overview** in your Mixpanel dashboard. Refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id">Mixpanel support guide</a> for more information.
```

File: docs/sources/extract/mixpanel.mdx [hunk 2]
```diff
@@ -28,22 +28,15 @@ To set up Mixpanel as a source in RudderStack, follow these steps:
...
 - **Attribution Window**: Specify the time period for attributing results to the ads and the lookback period during which the ads results are counted. By default, this value is set to **5 days**.
-- **Project Timezone**: Specify your Mixpanel project timezone by going to your Mixpanel **Project Settings** > **Project Details**.
-
-<div class="infoBlock">
-For more information on getting your Mixpanel project timezone, refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004547203-Manage-Timezones-for-Projects-in-Mixpanel">Mixpanel support guide</a>.
-</div>
+- **Project Timezone**: Specify your Mixpanel project timezone by going to your Mixpanel **Project Settings** > **Project Details**. For more information on getting your Mixpanel project timezone, refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004547203-Manage-Timezones-for-Projects-in-Mixpanel">Mixpanel support guide</a>.
```
Comment (ameypv-rudder) [inline docs/sources/extract/mixpanel.mdx:39]:
```suggestion
- **Project Timezone**: Specify your Mixpanel project timezone by going to **Project Settings** > **Project Details** in your Mixpanel dashboard. For more information on getting your Mixpanel project timezone, refer to the <a href="https://help.mixpanel.com/hc/en-us/articles/115004547203-Manage-Timezones-for-Projects-in-Mixpanel">Mixpanel support guide</a>.
```

File: docs/sources/extract/mixpanel.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/sources/extract/mixpanel.mdx:37]: LMK if this flow is accurate, or if you need to select the Mixpanel project first followed by going to these settings.

File: docs/sources/extract/mixpanel.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/sources/extract/mixpanel.mdx:39]: same as above.

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**